pkg/manifests/common.go

```diff
@@ -15,6 +16,24 @@ func GetTopLevelLabels() map[string]string { // this is a function to avoid any
 	return map[string]string{"app.kubernetes.io/managed-by": operatorName}
 }
 
+// Checks the first set of labels has the labels of the other passed in sets
+func HasTopLevelLabels(spcLabels map[string]string) bool {
```

Comment:
change the variable name. This is a generic function and used for more than spcLabels.